examples/setState/code-metadata.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import time
+from pathlib import Path
+from typing import Any, Dict, Sequence
+
+from multiversx_sdk_core import TokenComputer, AddressFactory
+from multiversx_sdk_core.transaction_factories import TransactionsFactoryConfig, TokenManagementTransactionsFactory, \
```

Comment:
Some imports can be removed, they seem unused.

Reply:
right, just copy-paste them in bulk

Reply:
removed unused imports